packages/fx-core/src/common/deps-checker/internal/funcToolChecker.ts

```diff
@@ -24,6 +25,26 @@ export enum FuncVersion {
   v4 = "4",
 }
 
+const FuncNodeVersionWhiteList: { [key: string]: { [key: string]: boolean } } = {
+  // func-core-tools version
+  "1": {
```

Comment:
Can we remove func v1 and v2 since we never support that...

Reply:
Removed.